api/v1alpha2/cloud/types.go

```diff
@@ -51,9 +51,22 @@ type Config struct {
 // ProviderConfig defines any extra information used to configure
 // the vSphere external cloud provider
 type ProviderConfig struct {
+	CCM CCMConfig `json:"ccmConfig,omitempty"`
```

Comment:
Hi @andrewsykim,

Please consider renaming field `CCM` to `Cloud` and field `CNS` to `Storage`.

Reply:
sgtm

Reply:
Going to use `CloudProvider` instead to avoid the package name stuter

Reply:
nvm, going to rename the package instead